ee/ucp/admin/configure/enable-audit-logging.md

```diff
@@ -1,99 +1,95 @@
+ignored list) and sent to Docker Engine through standard output (stdout). Creating audit logs is a UCP component that integrates with Swarm, Kubernetes, and UCP APIs.
 
 ## Logging levels
 
-To allow more control to administrators over the audit logging, three audit 
-logging levels are provided:
+The following levels are provided to give administrators more control over audit logging:
 
-- **None**: audit logging is disabled
+- **None:** Audit logging is disabled.
 
-- **Metadata**: includes the following:
+- **Metadata:** Includes the following:
     - Method and API endpoint for the request
     - UCP user who made the request
     - Response Status (success or failure)
     - Timestamp of the call
     - Object ID of any created or updated resource (for create or update API 
-    calls). We do not include names of created or updated resources
+    calls). The names of created or updated resources are not included.
     - License Key
     - Remote Address
 
-- **Request**: includes all fields from the Metadata level as well as the 
+- **Request:** Includes all fields from the Metadata level as well as the 
 request payload.
 
 > Note
 >
 > Once UCP audit logging has been enabled, audit logs can be found within the 
 > container logs of the `ucp-controller` container on each UCP manager node. 
-> Please ensure you have a 
-> [logging driver](../../../../config/containers/logging/configure/) 
-> configured appropriately with log rotation set as audit logging can start to 
-> generate a lot of data. 
+> Ensure that you have a [logging  driver](../../../../config/containers/logging/configure/) 
+> configured appropriately with log rotation set, as audit logging can start to 
+> generate a large amount of data. 
 
 ## Benefits
 
 You can use audit logs to help with the following use cases:
 
-- **Historical troubleshooting** - Audit logs are helpful in determining a 
+- **Historical troubleshooting:** Audit logs are helpful in determining a 
 sequence of past events that explain why an issue occurred.
 
-- **Security analysis and auditing** - Security is one of the primary uses for 
+- **Security analysis and auditing:** Security is one of the primary uses for 
 audit logs. A full record of all user interactions with the container 
 infrastructure gives your security team full visibility to questionable or 
 attempted unauthorized accesses.
 
-- **Chargeback** - You can use audit logs and information about the resources to 
+- **Chargeback:** You can use audit logs and information about the resources to 
 generate chargeback information.
 
-- **Alerting** - If there is a watch on an event stream or a notification 
+- **Alerting:** If there is a watch on an event stream or a notification 
 created by the event, alerting features can be built on top of event tools that 
-generate alerts for ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
+generate alerts for Ops teams (PagerDuty, OpsGenie, Slack, or custom solutions).
```

Comment:
In my opinion, the descriptions for the four benefit types are mostly superfluous. If you find that you agree, I would eliminate the bullet list and instead roll the benefits up into the first sentence:

"Audit logs can be of help with historical troubleshooting, security analysis and auditing, generating chargeback information, and creating useful alerts."